test/acceptance/util/Config.scala

```diff
@@ -25,7 +25,6 @@ object Config {
     logger.info(s"Stage: ${conf.getString("stage")}")
     logger.info(s"Subscriptions Frontend: ${baseUrl}")
     logger.info(s"Identity Frontend: ${identityFrontendUrl}")
-    logger.info(s"Screencast = https://saucelabs.com/tests/${Driver.sessionId}")
```

Comment:
This is just some tidy up that I missed when moving from SauceLabs to BrowserStack.